clang/lib/Analysis/FlowSensitive/DataflowEnvironment.cpp

```diff
@@ -290,13 +295,13 @@ widenKeyToValueMap(const llvm::MapVector<Key, Value *> &CurMap,
 namespace {
 
 // Visitor that builds a map from record prvalues to result objects.
-// This traverses the body of the function to be analyzed; for each result
-// object that it encounters, it propagates the storage location of the result
-// object to all record prvalues that can initialize it.
+// This traverses the AST sub-tree to be visited; for each result object that
+// it encounters, it propagates the storage location of the result object to all
+// record prvalues that can initialize it.
 class ResultObjectVisitor : public RecursiveASTVisitor<ResultObjectVisitor> {
 public:
   // `ResultObjectMap` will be filled with a map from record prvalues to result
-  // object. If the function being analyzed returns a record by value,
+  // object. If the sub-tree to be visited returns a record by value,
```

Comment:
Same confusion around analysis as above. I'm also having difficulty with the wording for this because this is a comment on the constructor, not a Visit/Traverse function. Tried another version.